package.json

```diff
@@ -89,6 +88,6 @@
     "node": ">=18.0.0"
   },
   "prisma": {
-    "seed": "ts-node -r tsconfig-paths/register prisma/seed.ts"
+    "seed": "ts-node-esm -r tsconfig-paths/register prisma/seed.ts"
```

Comment:
If we want to use 'tsx' this could also just be moved to it. So it would become:
```typescript
"seed": "tsx prisma/seed.ts"
```